server/paths/refer.ts

```diff
@@ -15,9 +15,14 @@ const newReferralProgrammeHistoryProgrammePath = newReferralProgrammeHistoryShow
 const newReferralConfirmOasysPath = newReferralShowPath.path('confirm-oasys')
 const newReferralAdditionalInformationPath = newReferralShowPath.path('additional-information')
 
+const caseListPath = referPathBase.path('case-list')
```

Comment:
Will this mean the path will be `/refer/case-list`, while Assess is `/assess/referrals/case-list`? I know we talked about the pathing elsewhere, but should this be consistent with Assessif we're going for that path for now?

Reply:
Yeah will make them consistent.  Will move this commit to the other PR though.